src/sidebarLearn.json

```
@@ -125,36 +125,36 @@
       ]
     },
     {
-      "title": "Managing State",
+      "title": "التحكم في الحالة",
       "path": "/learn/managing-state",
       "tags": ["intermediate"],
       "routes": [
         {
-          "title": "Reacting to Input with State",
+          "title": "الاستجابة للمدخلات باستخدام الحالة",
           "path": "/learn/reacting-to-input-with-state"
         },
         {
-          "title": "Choosing the State Structure",
+          "title": "اختيار هيكل الحالة",
           "path": "/learn/choosing-the-state-structure"
         },
         {
-          "title": "Sharing State Between Components",
+          "title": "مشاركة الحالة بين المكونات",
           "path": "/learn/sharing-state-between-components"
         },
         {
-          "title": "Preserving and Resetting State",
+          "title": "حفظ وإعادة تعيين الحالة",
           "path": "/learn/preserving-and-resetting-state"
         },
         {
-          "title": "Extracting State Logic into a Reducer",
+          "title": "استخلاص منطق الحالة إلى مخفض (reducer)",
           "path": "/learn/extracting-state-logic-into-a-reducer"
         },
         {
-          "title": "Passing Data Deeply with Context",
+          "title": "تمرير البيانات إلى عمق باستخدام السياق (context)",
           "path": "/learn/passing-data-deeply-with-context"
         },
         {
-          "title": "Scaling Up with Reducer and Context",
+          "title": "التوسع بواسطة المخفض (reducer) و السياق (context)",
           "path": "/learn/scaling-up-with-reducer-and-context"
         }
       ]
```

دعنا نؤجل ترجمة العناوين لحين ترجمة الصفحات حتى لا يحصل اختلاف بين العنوان هنا والعنوان في الصفحة
```suggestion
      "routes": [
        {
          "title": "Reacting to Input with State",
          "path": "/learn/reacting-to-input-with-state"
        },
        {
          "title": "Choosing the State Structure",
          "path": "/learn/choosing-the-state-structure"
        },
        {
          "title": "Sharing State Between Components",
          "path": "/learn/sharing-state-between-components"
        },
        {
          "title": "Preserving and Resetting State",
          "path": "/learn/preserving-and-resetting-state"
        },
        {
          "title": "Extracting State Logic into a Reducer",
          "path": "/learn/extracting-state-logic-into-a-reducer"
        },
        {
          "title": "Passing Data Deeply with Context",
          "path": "/learn/passing-data-deeply-with-context"
        },
        {
          "title": "Scaling Up with Reducer and Context",
          "path": "/learn/scaling-up-with-reducer-and-context"
        }
      ]
```